### PR TITLE
Add Lite documents in crosswalk-website.

### PR DESCRIPTION
**Pull request by heke123**
Crosswalk-Lite have been promoted as a stable release since the version
Lite-17. So we need to summarize the documents of Lite and update the
crosswalk-website.

BUG=XWALK-6774

### CODE REVIEW

**Comment by heke123** [comment]
@halton  @bspencer  Please have a look, thanks.

**Comment by heke123** [comment]
@halton,  The cordova part is not included. I'll updated that part in another PR after it completely done.

File: public/documentation/crosswalk_lite.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+* Uses a separate branch and has separate binary releases
+* Features which are cut off are evaluated closely by the team and users are encouraged to provide feedback
+* Code reviews and contribution model is the same as for Crosswalk
+* Currently the project does not follow canary/beta/stable channels like the main project.
+* Release cycles are 12-week - or on-request.
+
+## Way to reduce binary size
+
+* Cut feature([Crosswalk Lite Disabled feature list](/documentation/crosswalk_lite/lite_disabled_feature_list.html)). We use flags to enable/disable features like WebRTC/WebDatabase/WebGeolocation/... etc.
```
Comment (halton) [inline public/documentation/crosswalk_lite.md]: I'm afraid maintaining the disable list on website is not so convenient than we maintain it on wiki as we're doing now. I'd suggest  keep them there. @bspencer WDYT?

File: public/documentation/crosswalk_lite/use_crosswalk_lite.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                }
+            }
+
+### App Launched
+  The first time the application runs, the following UI dialog is displayed:
+  <p><img src='https://crosswalk-project.org/assets/crosswalk-lite-uncompress-dialog.png'></p>
+  It won't pop out anymore after the initialization finished successfully.
+  
+
+
```
Comment (halton) [inline public/documentation/crosswalk_lite/use_crosswalk_lite.md]: Remove above empty lines.

File: public/documentation/crosswalk_lite.md (hunk 1 above)
Comment (bspencer) [inline public/documentation/crosswalk_lite.md]: I agree it is a hassle except that I do like having the list on the website  :)   How often will it change without other changes to the website?

Other notes:
  What does "Enabled" mean and why would those items be in this list?
  Geolocation & Geofencing 	Disabled 	26K.   That seems like a common library, not worth 26K savings.

**Comment by bspencer** [comment]
Thanks for the PR.  I will review more thoroughly tomorrow and merge.  Has QA tested our builds and instructions?

"The first time the application runs, the following UI dialog is displayed: "
Does this happen at install or at first run?  If only on first run then is it decompressing the runtime and taking up additional space in storage from then on?  This means the APK is smaller, but not the installed app, right?  How much do we gain with the compression... is it worth it?

File: public/documentation/crosswalk_lite.md (hunk 1 above)
Comment (heke123) [inline public/documentation/crosswalk_lite.md]: Features are controlled by flags,  we found some features are needed by customer, so we set them as "Enabled" by default.  I agree the Geolocation feature is quite common, I'll set it as "Enabled" by default.

**Comment by heke123** [comment]
**What does "Enabled" mean and why would those items be in this list?**
----- Features are controlled by flags, we found some features are needed by customer, so we set them as "Enabled" by default. I agree the Geolocation feature is quite common, I'll set it as "Enabled" by default.
**Has QA tested our builds and instructions?**
----- QA has tested on Lite before it is promoted as stable release. 
**About the compression...**
----- Yes, you are right. APK is smaller, but not the installed app. More than 6M size is reduced, and it doesn't hurt any feature functions. I think it is a good solution for those people care more on the APK size.

 I'll make some modification base on your comments, thanks very much.

**Comment by heke123** [comment]
I found the "https://crosswalk-project.org/assets/crosswalk-lite-uncompress-dialog.png"  need to be updated, How can I upload a new picture to replace it? Thanks.

File: public/documentation/crosswalk_lite.md (hunk 1 above)
Comment (halton) [inline public/documentation/crosswalk_lite.md]: I received complain about Geolocation as well, I think it is time to re-enable it. https://crosswalk-project.org/jira/browse/XWALK-6785 is create for this.

Anyhow, the disable/enable list might be changed from time to time, we let's maintain them on wiki.

Comment (heke123) [inline public/documentation/crosswalk_lite.md]: No problem.

**Comment by bspencer** [comment]
We say:
> Lite is not released as often as mainline Crosswalk, nor it is rebased to the latest Chromium with the same cadence. Lite does not follow canary/beta/stable channels like the main project. Release cycles are 12-week - or on-request.

Can we agree upon:
 - exact release dates
 - For each release: 
     - have the specific features included/excluded listed on the website or wiki, not just the latest.
     - Have some QA indicator showing issues and QA validation status
     - Show the size of APK and installed footprint
 - Is there Cordova plugin support?  If not we should mention that

> ...or on request

Do we offer this as an option?  What would the request be?

Should we include the Lite builds on the Download page?

**Comment by bspencer** [comment]
What is the link to the Lite feature "disabled" list?

**Comment by bspencer** [comment]
"Runtime is preparing..." should be better as "Preparing runtime..."   If it takes more than 2 seconds, you could also write:  "<br>(This only happens once.)"

**Comment by halton** [comment]
> "Runtime is preparing..." should be better as "Preparing runtime..." If it takes more than 2 seconds, you could also write: "
(This only happens once.)"

Good suggestion, I think we need change the code, @heke123 could you please report a JIRA and fix in code. (I guess master code also need this change)

**Comment by heke123** [comment]
@halton  Yes, I'll report it in JIRA. The screenshot picture is not correct either. how to update the pic of "https://crosswalk-project.org/assets/crosswalk-lite-uncompress-dialog.png"?

**Comment by halton** [comment]
You need change the code and get re-make the screenshot.

**Comment by bspencer** [comment]
You can send a pull request or send me the picture and I’ll update it.

All assets are located in:  /public/assets/
Easy!  ☺

Bob

**Comment by heke123** [comment]
@bspencer  Thanks!  the cordova plugin is in testing, I'll send you the docs of cordova related later.
I'll make some update based on your comments.

**Pull request merged**